src/test/java/integration/SelenideMethodsTest.java

```diff
@@ -11,15 +11,55 @@
 import org.openqa.selenium.WebDriverException;
 import org.openqa.selenium.WebElement;
 
-import static com.codeborne.selenide.Condition.*;
```

Comment:
@andrejska @rosolko 
I don't like such changes. It replace few starts import to at least 50 new lines.
I find it outrageous.